marklogic-data-hub-central/ui/src/pages/TilesView.tsx

```diff
@@ -93,6 +93,10 @@ const TilesView = (props) => {
             setCurrentNode(props.id); // TODO Handle multiple with nested objects
             setOptions(tiles[props.id]);
             setView(views[props.id]);
+            if(props.id == 'explore')
```

Comment:
Do we still need setView(views[props.id]); before the if condition?

Reply:
@bsrikan I don't think so we need the setView before if condition

Reply:
Also what if we just call setView(views[props.id], true) instead of "if" condition?

Reply:
@bsrikan The boolean parameter holds true only for explorer tile, for all other tiles we don't need the second parameter hence we need that condition

Reply:
True and exactly for the same reason -- "zeroState: true", wont impact other tiles -- I thought we can achieve the behavior we want with 1 line change.

Reply:
@wooldridge I see that we are no longer using onSelect function in tilesview.tsx ? do we still need that for anything ?

Reply:
> True and exactly for the same reason -- "zeroState: true", wont impact other tiles -- I thought we can achieve the behavior we want with 1 line change.

@bsrikan right that works.

Reply:
@Sanjeevani19 I'll remove onSelect, thanks! I'm not sure about how we want to handle the setView at this point.